### ISSUE TITLE
Form Widget - HTML and Step components inherit the "Required" attribute, when cloned from any other form input

### ISSUE THREAD

**Issue by barrelmarketing** [elementor/elementor#15643]
Hi,

## Prerequisites
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

I discovered this while creating a form with Steps and HTML text labels. For some of my steps (and HTML labels), I cloned/duplicated existing text or number inputs that I had marked "Required", and set them to Step or HTML where appropriate.

We kept getting "Server Error" as the response on the form. Upon inspecting the response from admin-ajax.php, the server-side validator was throwing "Required" errors on our Step or HTML fields, despite these inputs not having a place for any input values.

## Steps to reproduce

1. Insert Elementor Form widget
2. Create text field, mark "Required"
3. Duplicate/clone text field, set to "Step" or "HTML" input
4. Ensure all form inputs have corresponding, unique IDs
4. Publish page, attempt to fill out form and submit.
5. Check admin-ajax.php for validation error.

## Isolating the problem
- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<html>
<body>
<!--StartFragment--><div class="elementor-system-info-section elementor-system-info-server" style="outline: 0px; margin-bottom: 10px; color: rgb(60, 67, 74); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Oxygen-Sans, Ubuntu, Cantarell, &quot;Helvetica Neue&quot;, sans-serif; font-size: 13px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(240, 240, 241); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Server Environment |   |  
-- | -- | --
Operating System: | Linux |  
Software: | Apache/2.4.25 (Debian) |  
MySQL version: | mariadb.org binary distribution v5.6.49-89.0 |  
PHP Version: | 7.3.28-2+0~20210604.85+debian9~1.gbp219f11 |  
PHP Max Input Vars: | 2500 |  
PHP Max Post Size: | 10M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected |  

</div><!--EndFragment-->
</body>
</html>
```
</details>

Thank you!

**Comment by Aviad-Herman** [comment]
Wow, great catch, thanks @barrelmarketing 
I will pass this on to the team

**Comment by Aviad-Herman** [comment]
Hey @barrelmarketing 
Are you still experience this issue on the latest versions?

**Comment by barrelmarketing** [comment]
> Hey @barrelmarketing
> Are you still experience this issue on the latest versions?

Hi - just created a test page on the newest version, looks like the team has fixed it.

Thank you!

**Comment by Aviad-Herman** [comment]
Perfect, thanks for raising this important issue!
